to_production.txt

```diff
@@ -1,31 +1,14 @@
-# New
-ofl/gajrajone # https://github.com/google/fonts/pull/5820
-ofl/gloock # https://github.com/google/fonts/pull/5749
-ofl/golostext # https://github.com/google/fonts/pull/5753
-ofl/labrada # https://github.com/google/fonts/pull/5799
-ofl/mynerve # https://github.com/google/fonts/pull/5743
-ofl/phudu # https://github.com/google/fonts/pull/5848
-ofl/shantellsans # https://github.com/google/fonts/pull/5785
-ofl/climatecrisis #https://github.com/google/fonts/pull/5350
-
 # Upgrade
-ofl/frankruhllibre # https://github.com/google/fonts/pull/5852
-ofl/gildadisplay # https://github.com/google/fonts/pull/5815
-ofl/martianmono # https://github.com/google/fonts/pull/5794
-ofl/rokkitt # https://github.com/google/fonts/pull/5800
-ofl/tirotelugu # https://github.com/google/fonts/pull/5772
-ofl/unicaone # https://github.com/google/fonts/pull/5807
-ofl/varelaround # https://github.com/google/fonts/pull/5823
+ofl/kumbhsans # https://github.com/google/fonts/pull/5667
 
 # Designer profile
 catalog/designers/agungrohmat # https://github.com/google/fonts/pull/5752
-catalog/designers/alexandrakorolkova # https://github.com/google/fonts/pull/5763
-catalog/designers/anyadanilova # https://github.com/google/fonts/pull/5788
-catalog/designers/arrowtype # https://github.com/google/fonts/pull/5788
-catalog/designers/duartepinto # https://github.com/google/fonts/pull/5751
-catalog/designers/saurabhsharma # https://github.com/google/fonts/pull/5821
-catalog/designers/shantellmartin # https://github.com/google/fonts/pull/5788
-catalog/designers/vitalykuzmin # https://github.com/google/fonts/pull/5806
 
 # Metadata / Description / License
-ofl/solway # https://github.com/google/fonts/pull/5849
+ofl/mingzat # https://github.com/google/fonts/pull/5819
+ofl/taiheritagepro # https://github.com/google/fonts/pull/5817
+
+# Axis Registry
+# https://github.com/google/fonts/pull/5853
+# FLARE, MORPH, OPSZ, VOLM, WDTH, WONK, XROT, YROT
```

Comment:
@chrissimpkins waiting for green light to add those to the prod list (cf https://github.com/google/fonts/pull/5853#issuecomment-1416294713)

Reply:
We should not push any axis registrations until Evan gives us the all clear.  I will let you know.